src/Locator/ExternalCommand.php

```diff
@@ -10,6 +10,8 @@
 
 class ExternalCommand
 {
+    private $suffixes = ['', '.phar'];
```

Comment:
This means we can drop the phar configure option for the symfony executable finder.
This list could be configurable. Don't know if that would add any value ATM though.

Reply:
We don't necessarily need to drop it. As for windows the executable would be found an iteration faster.

Reply:
No, you are right. it's ok